cmd/cli/version_test.go

```diff
@@ -180,37 +161,28 @@ func TestOutputVersionInfo(t *testing.T) {
 					},
 				},
 			},
-			expected:   "CLI Version: version.Info{Version:\"v0.0.0\", GitCommit:\"xxxxxxx\", BuildDate:\"0000-00-00-00:00\"}\nMesh [test] Version: version.Info{Version:\"v0.0.0\", GitCommit:\"xxxxxxx\", BuildDate:\"0000-00-00-00:00\"}\n",
+			expected:   "\nMESH NAME\tMESH NAMESPACE\tVERSION\tGIT COMMIT\tBUILD DATE\ntest\t\tv0.0.0\txxxxxxx\t0000-00-00-00:00\n",
 			clientOnly: false,
 		},
 		{
-			name: "cli and mesh versions with no remote version info",
-			versionInfo: versionInfo{
-				cliVersionInfo: &version.Info{
-					Version:   "v0.0.0",
-					GitCommit: "xxxxxxx",
-					BuildDate: "0000-00-00-00:00",
-				},
-				remoteVersionInfo: nil,
-			},
-			expected:   "CLI Version: version.Info{Version:\"v0.0.0\", GitCommit:\"xxxxxxx\", BuildDate:\"0000-00-00-00:00\"}\n",
-			clientOnly: false,
+			name:                  "cli and mesh versions with no remote version info",
+			remoteVersionInfoList: nil,
+			expected:              "\nMESH NAME\tMESH NAMESPACE\tVERSION\tGIT COMMIT\tBUILD DATE\n",
+			clientOnly:            false,
 		},
 	}
```

Comment:
I think it's worth adding a test case with multiple remote versions just to make sure newlines are being added as expected.